lib/keypair.ex

```diff
@@ -35,24 +35,25 @@ defmodule Elixium.KeyPair do
   end
 
   @doc """
-    Using a public address, fetch the correct keyfile and return the only the private key
+    Creates a new mnemonic to give to users based off private key
   """
-  @spec get_priv_from_file(String.t()) :: {binary, binary}
-  def get_priv_from_file(pub) do
-    unix_address = Application.get_env(:elixium_core, :unix_key_address)
-    key_path = unix_address <> "/" <> pub <> ".key"
-    {_pub, priv} = get_from_file(key_path)
-    priv
+  @spec create_mnemonic(binary) :: String.t()
+  def create_mnemonic(private) do
+    Mnemonic.from_entropy(private)
   end
 
-  @spec create_keyfile(tuple) :: :ok | {:error, any}
-  defp create_keyfile({public, private}) do
-    unix_address = Application.get_env(:elixium_core, :unix_key_address)
-    if !File.dir?(unix_address), do: File.mkdir(unix_address)
-
-    address = address_from_pubkey(public)
-
-    File.write(unix_address <> "/#{address}.key", private)
+  @doc """
+    Generates a keypair from the seed phrase or from the private key, leading " " will switch to mnemonic to import key from
+  """
+  @spec gen_keypair(String.t() | binary) :: {binary, binary}
+  def gen_keypair(phrase) do
+    case String.contains?(phrase, " ") do
```

Comment:
Since we're using a case statement here to match against true & false, I think we should use an if/else instead

Reply:
Fixed and updated